modules/node-observability-install-cli.adoc

```diff
@@ -49,7 +49,7 @@ metadata:
   namespace: node-observability-operator
 spec:
   targetNamespaces:
-  - node-observability-operator
+ []
```

Comment:
I believe the empty list should be in the same line as the key, so:
`targetNamespaces: []`